ScoreSaber/Core/ReplaySystem/UI/DesktopMainImberPanelView.cs

```diff
@@ -192,12 +182,15 @@ public void SetupObjects() {
                 y.transform.localPosition = new Vector3(0, -0.25f, 0);
             }
             tabSelector.transform.localScale = new Vector2(1.2f, 1.2f);
-            RecursivelyUnskewImageViews(_container.gameObject);
+            UnskewImageViews(_container.gameObject);
         }
 
+        [InjectOptional] private IFPFCSettings _fpfcSettings = null;
+
         [Inject]
         protected void Construct() {
-            if (!Plugin.FPFC) return;
+            if (_fpfcSettings == null) return;
+            if (!_fpfcSettings.Enabled && !Environment.GetCommandLineArgs().Contains("fpfc")) return; // fpfcSettings is being inconsistent?
```

Comment:
i think it may be getting fpfc settings enabled too early? im still not sure but this is the only instance of it being weird, which is why i didnt do it again anywhere else